### PR TITLE
Adds basic list of studios to the MIVS admin interface

### PR DESCRIPTION
**Pull request by RobRuana**

### CODE REVIEW

**Comment by RobRuana** [comment]
I'm going to merge this now, because it's totally isolated, and there are people actively waiting on it. But please review it and make suggestions. I'm happy to make changes as required!

File: mivs/templates/mivs_admin/studios.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      {% if studio.primary_contact.cellphone -%}
+        <br>
+        {{ studio.primary_contact.cellphone }}
+      {%- endif %}
+    </td>
+    <td>
+      {{ studio.website|url_to_link(is_relative=False) }}
+    </td>
+    <td>
+      {{ c.STUDIO_STATUS[studio.status] }}
```
Comment (kitsuta) [inline mivs/templates/mivs_admin/studios.html]: Hm... shouldn't this be doable with `studio.status_label`?

Comment (RobRuana) [reply mivs/templates/mivs_admin/studios.html]: Blerg, I keep forgetting about our convenience suffix properties 😢 

You are correct, of course!

**Comment by RobRuana** [comment]
Strong agree on the MIVS UI overhaul.

**Pull request merged**